lib/eldap/asn1/ELDAPv3.asn1

```diff
@@ -286,5 +286,16 @@ PasswdModifyRequestValue ::= SEQUENCE {
 PasswdModifyResponseValue ::= SEQUENCE {
      genPasswd        [0]     OCTET STRING OPTIONAL }
 
+-- LDAP Control Extension for Simple Paged Results Manipulation
+-- https://www.rfc-editor.org/rfc/rfc2696.txt
+-- controlType 1.2.840.113556.1.4.319
+
+PagedResultsControl ::= SEQUENCE {
+    size            INTEGER (0..maxInt),
+                            -- requested page size from client
+                            -- result set size estimate from server
+    cookie          OCTET STRING
+}
```

Comment:
In [RFC 2696](https://www.rfc-editor.org/rfc/rfc2696.txt) this record is called `realSearchControlValue` and `PagedResultsControl` is different.
Why rename it?  Changing the naming could make future updates impossible without breaking compatibility.

Reply:
A simple test case would be good to, if possible.

Reply:
@HansN 

> In [RFC 2696](https://www.rfc-editor.org/rfc/rfc2696.txt) this record is called `realSearchControlValue` and `PagedResultsControl` is different.
> Why rename it

Good question. I think because the original name `realSearchControlValue` didn't make much sense to me - no mention of paging and the weird "real" prefix...

I'll revert to the original name if you think it has maintainability advantages.

> A simple test case would be good to, if possible.

I'll look for an example test to duplicate - you're just looking for a test of the ASN decoding is that right?